gadget/start_gadgetfs_RaspiZeroW.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
sudo will work for both normal users and root users. Most people tend to forget to run it as root and wonder why it doesn't work :)